pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "mcp-hydrolix"
-version = "0.1.4"
+version = "0.1.5"
```

```suggestion
version = "0.1.4"
```
For this project, don't proactively increment version numbers outside of a release